src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackServiceTest.java

```diff
@@ -15,30 +22,110 @@
 import java.util.Map;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
 import static uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.model.Classification.EXCEPTION;
 
+@ExtendWith(MockitoExtension.class)
 class CreateCaseCallbackServiceTest {
 
     private static final String EVENT_ID = "createCase";
-    private static final CreateCaseCallbackService SERVICE = new CreateCaseCallbackService(
-        new CreateCaseValidator()
-    );
+    private static final String SERVICE = "service";
+    private static final String CASE_TYPE_ID = SERVICE + "_ExceptionRecord";
+    private static final CreateCaseValidator VALIDATOR = new CreateCaseValidator();
+
+    @Mock
+    private ServiceConfigProvider serviceConfigProvider;
+
+    private CreateCaseCallbackService service;
+
+    @BeforeEach
+    void setUp() {
+        service = new CreateCaseCallbackService(
+            VALIDATOR,
+            serviceConfigProvider
+        );
+    }
 
     @Test
     void should_not_allow_to_process_callback_in_case_wrong_event_id_is_received() {
-        Either<List<String>, ExceptionRecord> output = SERVICE.process(null, "some event");
+        Either<List<String>, ExceptionRecord> output = service.process(null, "some event");
 
         assertThat(output.isLeft()).isTrue();
         assertThat(output.getLeft()).containsOnly("The some event event is not supported. Please contact service team");
+        verify(serviceConfigProvider, never()).getConfig(anyString());
+    }
+
+    @Test
+    void should_not_allow_to_process_callback_in_case_wrong_case_type_id_is_missing() {
+        // given
+        CaseDetails caseDetails = TestCaseBuilder.createCaseWith(builder -> builder.id(1L));
+
+        // when
+        Either<List<String>, ExceptionRecord> output = service.process(caseDetails, EVENT_ID);
+
+        assertThat(output.isLeft()).isTrue();
+        assertThat(output.getLeft()).containsOnly("No case type ID supplied");
+        verify(serviceConfigProvider, never()).getConfig(anyString());
+    }
+
+    @Test
+    void should_not_allow_to_process_callback_in_case_wrong_case_type_id_is_empty() {
```

Comment:
same as above

Reply:
do we need another test for wrong case type id?

Reply:
"wrong case type id" is covered with both: empty case type id and the unit test which asserts "Transformation URL is not configured" message.

any string is firstly asserting the correct suffix and then returns any service (as a prefix of case type) found. then it's down to service config provider to determine service is configured or not. so both - incorrect case type id and transformation url branches are covered